### PR TITLE
Update dependencies. r=azasypkin

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by gmarty** [comment]
@azasypkin: This patch is ready for review. What do you think?

File: package.json [hunk 1]
```diff
@@ -81,17 +81,17 @@
...
-    "run-sequence": "^1.2.0",
+    "run-sequence": "^1.2.1",
     "rxjs": "^5.0.0-beta.8",
     "selenium-webdriver": "^2.53.2",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
     "stylelint": "^6.5.1",
     "stylelint-config-standard": "^8.0.0",
-    "uglify-js": "mishoo/UglifyJS2#harmony"
+    "uglify-js": "github:mishoo/UglifyJS2#harmony"
```
Comment (azasypkin) [inline package.json]: Heh, looks much more obvious! Just curious, what troubles could old format cause?

Comment (gmarty) [inline package.json]: Nothing at all, but I ran a npm update or install and it normalised this repo for me, so I thought I'd keep it.

Comment (azasypkin) [inline package.json]: Right, I like it better, at least it's clear that it is github dependency :)

**Comment by azasypkin** [comment]
r=me, looks good, thanks for the update! I see that few more packages already have newest version, could you please bump up them as well?
* gulp-live-server 0.0.30
* rollup 0.30.0 \0/

**Pull request merged**